script/gen-theme.js

```diff
@@ -0,0 +1,172 @@
+var fs = require("fs");
+
+// name: [light, dark, black]
+
+var themeVariables = {
```

Comment:
`let` and `const` are available in js files & have been in node for a long time.